### PR TITLE
[ImageBuilder]Init of ImageBuilderTemplate.

### PR DESCRIPTION
**Pull request by wyunchi-ms**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ImageBuilder/custom/New-AzImageBuilderCustomizer.ps1 [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+Describes a unit of image customization
+.Description
+Describes a unit of image customization
+
+.Link
+https://docs.microsoft.com/en-us/powershell/module/az.imagebuilder/new-AzImageBuilderCustomizer
+#>
+function New-AzImageBuilderCustomizer {
+    [OutputType('Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20200214.IImageTemplateCustomizer')]
+    [CmdletBinding(PositionalBinding=$false, SupportsShouldProcess, ConfirmImpact='Medium', DefaultParameterSetName="ShellCustomizer")]
```
Comment (dolauli) [inline src/ImageBuilder/custom/New-AzImageBuilderCustomizer.ps1:26]: For pure client side operation without changing backend status/data, I do not think we need to support "shouldprocess"

Comment (wyunchi-ms) [reply src/ImageBuilder/custom/New-AzImageBuilderCustomizer.ps1]: Removed `ShouldProcess`

File: src/ImageBuilder/custom/Update-AzImageBuilderTemplate.ps1 [hunk 2]
```diff
@@ -0,0 +1,331 @@
+
+# # ----------------------------------------------------------------------------------
```
Comment (dolauli) [inline src/ImageBuilder/custom/Update-AzImageBuilderTemplate.ps1:2]: Remove this file if it is not required

**Pull request merged**